src/App.js

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import Title from './TitleComponent/TitleComponent'
+import './App.css'
+class App extends React.Component {
+
+    constructor(props){
+        super(props);
+        this.state = {
+            content : '',
+            name: '',
+            number: '',
+        }
+    }
+
+    inputHandler(event) {
+        event.persist();
+        this.setState((previousState)=>{
+            let newState = {...previousState};
+            newState.content = event.target.value;
+            return newState;
+        });
+    }
+
+    submitHandler(event) {
```

Comment:
The same, but here you can use handleSubmit or handleClick as a name

Reply:
changed name as per naming convention